lib/ebay_client/version.rb

```diff
@@ -1,3 +1,3 @@
 module EbayClient
-  VERSION = '0.2.0.981'
+  VERSION = '0.2.0.982'
```

Comment:
Sorry for the slow feedback, but how would just setting the token allow for "multiple users"? Aren't `appid`, `devid` and `certid` equally part of a user? I reckon that having several instances of the client each with it's own set of settings in an individual configuration would be more sensible: (see `engine.rb`)
```
  configurations = EbayClient::Configuration.load File.load('<config file>')
  configuration = configurations[<environment>]

  EbayClient.api = EbayClient::Api.new configuration
```